pkgs/by-name/ef/efmt/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "efmt";
+  version = "0.18.2";
+
+  src = fetchFromGitHub {
+    owner = "sile";
+    repo = pname;
```

Comment:
usage of pname is considered anti-pattern https://github.com/NixOS/nixpkgs/issues/277994